src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/IssueDocumentHandler.java

```diff
@@ -160,6 +161,21 @@ protected PreSubmitCallbackResponse<SscsCaseData> issueDocument(Callback<SscsCas
         return response;
     }
 
+    protected String getDocumentTypeLabel(SscsCaseData caseData, DocumentType documentType, String documentTypeLabel) {
+
+        String embeddedDocumentTypeLabel = (FINAL_DECISION_NOTICE.equals(documentType) ? "Decision Notice" : documentTypeLabel);
+
+        PostHearing postHearing = caseData.getPostHearing();
+        if (nonNull(postHearing) && nonNull(postHearing.getSetAside().getAction())) {
+            CcdCallbackMap action = postHearing.getSetAside().getAction();
+            if (action.toString().equals(SetAsideActions.GRANT.getCcdDefinition())
```

Comment:
```suggestion
            
            boolean isGrantOrRefuseSetAsideAction = action.toString().equals(SetAsideActions.GRANT.getCcdDefinition())
                || action.toString().equals(SetAsideActions.REFUSE.getCcdDefinition());
```
and then 
```suggestion
embeddedDocumentTypeLabel =  isGrantOrRefuseSetAside ? "Set Aside Decision Notice" : embeddedDocumentTypeLabel 
```

so instead of this :
```
if (nonNull(postHearing) && nonNull(postHearing.getSetAside().getAction())) {
            CcdCallbackMap action = postHearing.getSetAside().getAction();
            if (action.toString().equals(SetAsideActions.GRANT.getCcdDefinition())
            || action.toString().equals(SetAsideActions.REFUSE.getCcdDefinition())) {
                embeddedDocumentTypeLabel = "Set Aside Decision Notice";
            }
        }
        return embeddedDocumentTypeLabel;
```
```suggestion
if (nonNull(postHearing) && nonNull(postHearing.getSetAside().getAction())) {
     CcdCallbackMap action = postHearing.getSetAside().getAction();
     embeddedDocumentTypeLabel =  isGrantOrRefuseSetAsideAction ? "Set Aside Decision Notice" : embeddedDocumentTypeLabel;  
        }
        return embeddedDocumentTypeLabel;
```

Reply:
Updated as per the review comment.